shared/actions/gregor.js

```diff
@@ -146,12 +146,17 @@ function* _handlePushState(pushAction: GregorGen.PushStatePayload): SagaGenerato
 }
 
 function* handleKbfsFavoritesOOBM(kbfsFavoriteMessages: Array<OutOfBandMessage>): Generator<any, void, any> {
-  const msgsWithParsedBodies = kbfsFavoriteMessages.map(m => ({...m, body: JSON.parse(m.body.toString())}))
-  const createdTLFs = msgsWithParsedBodies.filter(m => m.body.action === 'create')
-
-  const username: string = (yield Saga.select(usernameSelector): any)
+  const createdTLFs: Array<{action: string, tlf: ?string}> = kbfsFavoriteMessages
```

Comment:
flow got confused when we overwrote the body prop with a new body. we don't even use the raw ...m part so lets just get rid of it